src/CosmosEditorManager.ts

```diff
@@ -49,7 +49,7 @@ export class CosmosEditorManager {
             }
         }
         this.fileMap[localDocPath] = editor;
-        const fileMapLabels = {};
+        const fileMapLabels = this._globalState.get(this._persistedEditorsKey);
```

Comment:
Hahaha didn't we just learn that this can return undefined? 🙃

You could just do `const fileMapLabels = this._globalState.get(this._persistedEditorsKey) || {};`

Reply:
Yep, you should think about stuff like this always.

BTW, have you searched for other uses of this._globalState.get() that might cause problems?

Reply:
@EricJizbaMSFT I'm going with [get<T>(key: string, defaultValue: T): T](https://code.visualstudio.com/docs/extensionAPI/vscode-api#Memento) - that looks cleaner and more official to me. 

@StephenWeatherford - Looked at issues filed and StackOverflow, I don't see anything surprising so far.

Reply:
@PrashanthCorp I was referring to if you've checked all uses of globalState.get() in our code...